go.mod

```diff
@@ -0,0 +1,36 @@
+module github.com/CanonicalLtd/serial-vault
+
+go 1.13
+
+require (
+	github.com/Masterminds/squirrel v1.2.0
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/godbus/dbus v4.1.0+incompatible // indirect
+	github.com/gorilla/csrf v1.0.3-0.20161122164500-69581736821c
```

Comment:
In `vendor.json` there was `"versionExact": "v1.5"` - how come the change?

I suspect there are other differences like this but I'm not doing a full audit. If they're necessary/reasonable/ok then sure, but provide context in e.g. the commit message or something please

Reply:
`v1.5` is not a valid SemVer as specified in the Go mod spec (it need to be `v1.5.0`) so it will be somehow translated into the date and hash value of the latest commit from this tag.

It's the same version, I double-checked every direct dependency.
https://github.com/gorilla/csrf/releases/tag/v1.5 has the hash ` 6958173` and the date is `Nov 22, 2016` (this is the date and hash from the latest commit in this version).

Reply:
From the vendor.json: the `revision` and `revisionTime` is the same as in go.mod. So the same git commit is pinned here and there.
```
		{
			"checksumSHA1": "2ieXR7YGRBDmQWVeUpjk5ay8iCE=",
			"path": "github.com/gorilla/csrf",
			"revision": "69581736821c33d85bbf378f42f6ad864dbd85de",
			"revisionTime": "2016-11-22T16:45:00Z",
			"version": "v1",
			"versionExact": "v1.5"
		},
```

Reply:
Makes sense, cheers - want to actually put this in the change so it's more visible for anyone else?

Reply:
OK, I will put this change into ChangeLog file and then in release notes in GitHub

Reply:
Added a note in the ChangeLog file